eng/Versions.props

```diff
@@ -10,12 +10,14 @@
     <MicrosoftDotNetGenApiPackage>Microsoft.DotNet.GenAPI</MicrosoftDotNetGenApiPackage>
     <MicrosoftDotNetGenFacadesPackage>Microsoft.DotNet.GenFacades</MicrosoftDotNetGenFacadesPackage>
     <MicrosoftDotNetXUnitExtensionsPackage>Microsoft.DotNet.XUnitExtensions</MicrosoftDotNetXUnitExtensionsPackage>
+    <MicrosoftDotNetBuildTasksPackagingPackage>Microsoft.DotNet.Build.Tasks.Packaging</MicrosoftDotNetBuildTasksPackagingPackage>
```

Comment:
I know you are just following convention but we might want to reconsider the package name duplication now if the only place the name is used is in Tools.props.